Amazon.QLDB.Driver.Tests/AsyncBufferedResultTests.cs

```diff
@@ -0,0 +1,88 @@
+﻿/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance with
+ * the License. A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+
+namespace Amazon.QLDB.Driver.Tests
+{
+    using System.Collections.Generic;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Amazon.IonDotnet.Tree;
+    using Amazon.IonDotnet.Tree.Impl;
+    using Microsoft.VisualStudio.TestTools.UnitTesting;
+    using Moq;
+    using static Amazon.QLDB.Driver.AsyncBufferedResult;
```

Comment:
We need this?

Reply:
We need either this import or `AsyncBufferedResult.BufferResultAsync()` (line 54) and `AsyncBufferedResult.ValuesAsyncEnumerator()` (line 51) .